Monika After Story/game/zz_windowreacts.rpy

```diff
@@ -320,6 +321,101 @@ label display_notif(title, body, group=None, skip_checks=False):
 
 
 #START: Window Reacts
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="notif_pinterest",
+            category=['pinterest'],
+            rules={"skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label notif_pinterest
+    call display_notif(m_name,"Something new today?",'Window Reactions')
+
+    #Unlock again if we failed
+    if not _return:
+        $ mas_unlockFailedWRS('monika_pinterest')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="notif_duolingo",
+            category=['duolingo'],
+            rules={"skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label notif_duolingo
+    call display_notif(m_name,"How is it with your progress in language, [player]?",'Window Reactions')
+
+    #Unlock again if we failed
+    if not _return:
+        $ mas_unlockFailedWRS('notif_duolingo')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="notif_wikipedia",
+            category=['wikipedia'],
+            rules={"skip alert": None},
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label notif_wikipedia
+    call display_notif(m_name,"Wanna learn something new?",'Window Reactions')
+
+    #Unlock again if we failed
+    if not _return:
+        $ mas_unlockFailedWRS('notif_wikipedia')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="reddit_rule34",
```

Comment:
One thing about the other one was that it was supposed to be *specifically* for Monika related searches, and not general. The expressions for the `show`s here don't really work if you're potentially looking at one of the others.

Reply:
> One thing about the other one was that it was supposed to be specifically for Monika related searches, and not general. The expressions for the shows here don't really work if you're potentially looking at one of the others.

I got it. I just tried to make it for Reddit version.